oci/oci.go

```diff
@@ -689,7 +689,18 @@ func (r *Runtime) StopContainer(ctx context.Context, c *Container, timeout int64
 
 	if timeout > 0 {
 		if err := utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.Path(c), "kill", c.id, c.GetStopSignal()); err != nil {
-			return fmt.Errorf("failed to stop container %s, %v", c.id, err)
+			process, perr := findprocess.FindProcess(c.state.Pid)
+			if perr == findprocess.ErrNotFound {
+				c.state.Finished = time.Now()
+				return nil
+			}
+			if perr == nil {
+				err = process.Release()
+				if err != nil {
+					logrus.Warnf("failed to release process %d for container %q: %v", c.state.Pid, c.ID(), err)
```

Comment:
With `c.id` working here (and throughout this package), we probably want to replace `c.ID()` with `c.id` here.  Even if the Go compiler optimizes those function calls away, we benefit by saving a few characters in the source.

If, on the other hand, the preference is to always use `c.ID()`, even in this package (to make future migrations where we move `c.id` easier?), we should update the `Errorf` below to use `c.ID()`.

Reply:
future cleanup is going to be ok ;)

Reply:
> future cleanup is going to be ok ;)

Is there a reason to punt on this?  It's a tiny change here, and you just bumped the PR a few minutes ago for [this][1] anyway.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1478#discussion_r178847575

Reply:
it's not part of this PR, and it's not just 1-2 occurrences, if you'd like to do this, and I'd love to see it, let's create another PR

Reply:
> it's not part of this PR...

Ah, for this PR, I was thinking just the new lines you're adding.

Reply:
I've spun this off into #1487.